nifi-stateless/nifi-stateless-core/src/test/java/org/apache/nifi/stateless/core/InitFromFlowXmlIT.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.stateless.core;
+
+
+//import com.google.gson.JsonObject;
+//import com.google.gson.JsonParser;
+//import org.apache.nifi.bundle.Bundle;
+//import org.apache.nifi.bundle.BundleCoordinate;
+//import org.apache.nifi.controller.exception.ProcessorInstantiationException;
+//import org.apache.nifi.controller.serialization.FlowEncodingVersion;
+//import org.apache.nifi.controller.serialization.FlowFromDOMFactory;
+//import org.apache.nifi.encrypt.StringEncryptor;
+//import org.apache.nifi.nar.ExtensionManager;
+//import org.apache.nifi.nar.NarUnpacker;
+//import org.apache.nifi.parameter.Parameter;
+//import org.apache.nifi.parameter.ParameterContext;
+//import org.apache.nifi.registry.VariableDescriptor;
+//import org.apache.nifi.reporting.InitializationException;
+//import org.apache.nifi.stateless.bootstrap.ExtensionDiscovery;
+//import org.apache.nifi.util.NiFiProperties;
+//import org.apache.nifi.web.api.dto.ProcessGroupDTO;
+//import org.glassfish.jersey.internal.jsr166.Flow;
+//import org.w3c.dom.Document;
+//import org.w3c.dom.Element;
+//
+//import javax.net.ssl.SSLContext;
+//import java.io.File;
+//import java.io.FileNotFoundException;
+//import java.io.IOException;
+//import java.net.MalformedURLException;
+//import java.net.URL;
+//import java.net.URLClassLoader;
+//import java.nio.file.Path;
+//import java.nio.file.Paths;
+//import java.util.*;
+//
+//import static org.mockito.ArgumentMatchers.anyString;
+//import static org.mockito.Mockito.mock;
+//import static org.mockito.Mockito.when;
+
+public class InitFromFlowXmlIT {
+
+//    private NiFiProperties getNiFiProperties() {
```

Comment:
Whole body is commented out.

Reply:
This was failing when trying to load the processor classes from the standard processors nar. I think it has to do with the narloader but I wasn't sure where to go from there. I'll remove this and add unit tests for the flowxml parser instead

Reply:
Added FlowXml parser test